extensions/wikia/ExactTargetUpdates/tests/tasks/ExactTargetWikiTaskTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class ExactTargetWikiTaskTest extends WikiaBaseTest {
+
+	public function setUp() {
+		$this->setupFile = __DIR__ . '/../../ExactTargetUpdates.setup.php';
+		parent::setUp();
+	}
+
+	/**
+	 * @expectedException \Wikia\Util\AssertionException
+	 * @expectedExceptionMessage Wiki ID missing
+	 */
+	public function testShouldThrowExceptionWhenUpdatingWikiWithoutWikiId() {
+		$clientMock = $this->prepareClientMock();
+		$task = $this->getWikiTask($clientMock);
+
+		$task->updateWiki( null );
+	}
+
+	public function testShouldCallClientUpdateWikiMethodWithProperParams() {
+		$clientMock = $this->prepareClientMock();
+		$task = $this->getWikiTask($clientMock);
+
+		$clientMock->expects( $this->once() )
+			->method( 'updateWiki' );
+
+		$task->updateWiki( 1 );
+	}
+
+	private function getWikiTask($client) {
+		return new \Wikia\ExactTarget\ExactTargetWikiTask($client);
+	}
+
+	/**
+	 * @return PHPUnit_Framework_MockObject_MockObject
+	 */
+	private function prepareClientMock() {
+		$mockClient = $this->getMock( '\Wikia\ExactTarget\ExactTargetClient',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$mockClient". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1488)